doc/index.md

```diff
@@ -687,6 +709,34 @@ Params:
 
 Returns: void
 
+#### `isPersistentConnectionEnabled()`
+Check to see if the user's currently associated station has
+persistent connection enabled.
+
+Declaration:
+``` ts
+isPersistentConnectionEnabled(): boolean;
+```
+
+Params: none
+
+Returns: a boolean of if the station has persistent connection enabled
```

Comment:
I think a better way to word this would be - "Returns: a boolean, `true`, if the station has persistent connection enabled."

Reply:
Thank you! I could not figure out a good way to say this.